src/main/java/life/qbic/openbis/openbisclient/OpenBisClient.java

```diff
@@ -1051,6 +1055,15 @@ public Vocabulary getVocabulary(String vocabularyCode) {
     }
   }
 
+  @Override
+  public void ingest(String dss, String serviceName, Map<String, Object> params) {
+    ServiceFinder serviceFinder2 =
+        new ServiceFinder("openbis", IQueryApiServer.QUERY_PLUGIN_SERVER_URL);
+    IQueryApiServer openbisDssService = serviceFinder2.createService(IQueryApiServer.class,
+        this.url);
+    openbisDssService.createReportFromAggregationService(this.sessionToken, dss, serviceName,
```

Comment:
Just for my understanding: does this method also register/ingest the params in openbis through the ServiceFinder?

Reply:
yes